usaspending_api/database_scripts/job_archive/management/commands/generate_cares_act_test_data.py

```diff
@@ -0,0 +1,297 @@
+        )
+
+        parser.add_argument(
+            "fiscal quarter", type=self.validate_fiscal_quarter, help='Fiscal quarter to be "enhanced".',
+        )
+
+        parser.add_argument(
+            "--yes-i-know-its-rds",
+            action="store_true",
+            help="Just a safety precaution.  This switch is required to run against an RDS instance.",
+        )
+
+        parser.add_argument(
+            "--vacuum", action="store_true", help="If you'd like the script to perform a little housekeeping for you.",
+        )
+
+        parser.epilog = (
+            "WARNING!  THIS SCRIPT WILL IRREPARABLY MODIFY SUBMISSIONS AND FILE A/B/C DATA IN THE TARGET "
+            "DATABASE!  DO NOT RUN ON PRODUCTION OR ANY DATABASE YOU CANNOT EASILY RESTORE!"
+        )
+
+    def handle(self, *args, **options):
+        with ScriptTimer("CARES Act test data generation"):
+
+            self.set_state(options)
+            self.perform_validations()
+
+            with transaction.atomic():
+                self.record_base_submission_ids()
+                self.clone_submissions()
+                self.update_submissions()
+                self.clone_for_defc()
+
+                t = ScriptTimer("Commit transaction")
+            t.log_success_message()
+
+            if self.vacuum:
+                self.vacuum_tables()
+
+    def set_state(self, options):
+        self.fiscal_year = options["fiscal year"]
+        self.fiscal_quarter = options["fiscal quarter"]
+        self.allow_rds = options["yes_i_know_its_rds"]
+        self.vacuum = options["vacuum"]
+
+        # Some other nice-to-haves we'll be using in loops.
+        self.fiscal_period = {1: 3, 2: 6, 3: 9, 4: 12}[self.fiscal_quarter]
+        self.clone_periods = {1: [2], 2: [4, 5], 3: [7, 8], 4: [10, 11]}[self.fiscal_quarter]
+
+        # This adds some variability to clones.  Ratios for periods in the same quarter should add up to 1.
+        # If they don't, no big deal really, but it keeps the dollar figures in line across Files A, B, and C.
+        self.period_ratios = {2: 0.3, 3: 0.7, 4: 0.1, 5: 0.2, 6: 0.7, 7: 0.1, 8: 0.2, 9: 0.7, 10: 0.1, 11: 0.2, 12: 0.7}
+
+        self.period_starts = {
+            p + 1: (date(self.fiscal_year - 1, 10 + p, 1) if p < 3 else date(self.fiscal_year, p - 2, 1))
+            for p in range(13)
+        }
+        self.period_ends = {p + 1: self.period_starts[p + 2] - timedelta(days=1) for p in range(12)}
+
+        # There is no fiscal period 1 and fiscal period 2 starts where fiscal period 1 would have.
+        self.period_starts[2] = self.period_starts[1]
+        del self.period_starts[1]
+        del self.period_ends[1]
+
+    def perform_validations(self):
+        if self.allow_rds is False and self.is_rds():
+            raise RuntimeError(
+                "You appear to be running against an RDS instance.  Consider using the --yes-i-know-its-rds "
+                "switch if you no longer value your career."
```

Comment:
```suggestion
                "You appear to be running against an RDS instance.  Please note  that this will permanently change a large amount of this remote instance's data. If you are sure you want to do this, use the --yes-i-know-its-rds flag"
```

I definitely think we should run this on DEV sooner rather than later BTW

Reply:
Definitely, but I'd like to see at least a couple other people run it locally first to make sure nothing bursts into flames.

Reply:
I ran it locally and in Sandbox. No issues observed